app/lib/features/onboarding/providers/notifiers/auth_notifier.dart

```diff
@@ -89,31 +81,22 @@ class AuthStateNotifier extends StateNotifier<bool> {
   Future<void> logout(BuildContext context) async {
     final sdk = await ref.read(sdkProvider.future);
     final stillHasClient = await sdk.logout();
-    final loggedInNotifier = ref.read(isLoggedInProvider.notifier);
-    final clientNotifier = ref.read(clientProvider.notifier);
     if (stillHasClient) {
       debugPrint('Still has clients, dropping back to other');
-      loggedInNotifier.update((state) => true);
-      ref.invalidate(clientProvider);
+      ref.read(clientProvider.notifier).state = sdk.currentClient;
       ref.invalidate(spacesProvider);
-      clientNotifier.state = sdk.currentClient;
```

Comment:
Great refactoring

Reply:
And we may want to replace `List<Client> _clients = []` with `Client? _currentClient = null` in `acter_flutter_sdk.dart`.

Reply:
Nope, we generally want multi-client-support on the ffi-layer.